pyqldb/communication/session_client.py

```diff
@@ -11,8 +11,6 @@
 
 from logging import getLogger
 
-from amazon.ion.simple_types import IonPyBool, IonPyBytes, IonPyDecimal, IonPyDict, IonPyFloat, IonPyInt, IonPyList, \
```

Comment:
Is this related to this change?

Reply:
Nope, reverted this change, I will put it as part of another PR